README.md

```diff
@@ -1,38 +1,77 @@
-## Welcome!
-Integrating passkey technology can be really hard. That's why we built the Passage Flutter SDK - to make passkey authentication easy for you and your users - across native iOS, native Android, and web.
+## About
 
-<img width="1069" alt="Passage Flutter" src="https://storage.googleapis.com/passage-docs/passage-flutter.png">
+[Passage by 1Password](https://1password.com/product/passage) unlocks the passwordless future with a simpler, more secure passkey authentication experience. Passage handles the complexities of the [WebAuthn API](https://blog.1password.com/what-is-webauthn/), and allows you to implement passkeys with ease.
 
-<br>
+Use [Passkey Flex](https://docs.passage.id/flex) to add passkeys to an existing authentication experience.
 
-## Installation
+Use [Passkey Complete](https://docs.passage.id/complete) as a standalone passwordless auth solution.
 
-```sh
+Use [Passkey Ready](https://docs.passage.id/passkey-ready) to determine if your users are ready for passkeys.
+
+### In passage-flutter
+
+Use passage-flutter to implement Passkey Complete into your Flutter backend to authenticate requests and manage users.
+
+| Product                                                                                                                                  | Compatible                                                                                                                                                                                                                |
+| ---------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Passkey Flex](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-flex-icon.png) Passkey **Flex**             | ✖️ For Passkey Flex, check out the [Passkey Flex APIs](https://docs.passage.id/flex/apis)                                                                                                                                 |
+| ![Passkey Complete](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-complete-icon.png) Passkey **Complete** | ✅                                                                                                                                                                                                                        |
+| ![Passkey Ready](https://storage.googleapis.com/passage-docs/github-md-assets/passage-passkey-ready-icon.png) Passkey **Ready**          | ✖️ For Passkey Ready, check out [Authentikit for Android](https://github.com/passageidentity/passage-android/tree/main/authentikit) and [Authentikit for iOS](https://github.com/passageidentity/passage-authentikit-ios) |
```

Comment:
Do we want to link to mobile-specific versions of Authentikit here?

Reply:
Looks great!